WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -441,4 +443,62 @@ public static boolean currentUserCanDeleteMedia(@NonNull SiteModel site) {
     public static int getFlingDistanceToDisableThumbLoading(@NonNull Context context) {
         return ViewConfiguration.get(context).getScaledMaximumFlingVelocity() / 2;
     }
+
+    /*
+     * returns a MediaModel from a device media URI
+     */
+    public static MediaModel mediaModelFromLocalUri(@NonNull Context context,
```

Comment:
I think this method belongs to / would be a great addition to `FluxCUtils.java` in the FluxC library. Can we move it there? We can open a separate issue as well

Reply:
Sure, we can move that to FluxC. I'd prefer that as a separate issue, though.

Reply:
👍